opynions/pytest_opynions.py

```diff
@@ -0,0 +1,109 @@
+# -*- coding: utf-8 -*-
+"""
+pytest plugin to test if specific repo follows standards
+"""
+import os
+from collections import defaultdict
+
+import pytest
+
+import yaml
+
+session_data_holder_dict = defaultdict(dict)
+
+@pytest.fixture(scope="session")
+def all_results():
+    return session_data_holder_dict
+
+def pytest_configure(config):
+    """
+    pytest hook used to add pytest_opynions install dir as place for pytest to find tests
+    """
+
+    if config.getoption("repo_health_check"):
+
+        # Add path to pytest-opynions dir so pytest knows where to look for checks
+        file_dir = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
+        config.args.append(file_dir)
+
+        # add repo path so a repo can design their own checks inside a repo_state_checks dir
```

Comment:
We should be consistent with naming.  Should we call this directory "opynions"?  Should both the package and the directory use a different name instead?

Reply:
names have been changed.